externs/shaka/player.js

```diff
@@ -1238,6 +1239,10 @@ shaka.extern.StreamingConfiguration;
  * @description
  *   Media source configuration.
  *
+ * @property {shaka.config.CodecSwitchingStrategy} codecSwitchingStrategy
+ *   Allow codec switching strategy. Default behavior is disabled.
```

Comment:
The default behavior is not disabled. You commented out the code that handled that, and do not have disabled as an option on the enum. It defaults to SMOOTH if smooth codec switching is supported, RELOAD overwise.

Reply:
Changed!